[DualTor][Mellanox] Skip dualtor test_downstream_ecmp_nexthops test on Mellanox Spectrum platforms

congh-nvidia opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/sonic-net/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
The hierarchy ecmp(re-hash) behaviour is not as expected in the dualtor test test_downstream_ecmp_nexthops when running on Spectrum devices unlike in Spectrum-2 and above devices.

### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)


### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

### Approach
#### What is the motivation for this PR?
Skip dualtor test_downstream_ecmp_nexthops test on Mellanox Spectrum platforms

#### How did you do it?
Add the skip in tests_mark_conditions.yaml

#### How did you verify/test it?
By automation.

#### Any platform specific information?
Only for Mellanox Spectrum platforms

#### Supported testbed topology if it's a new test case?

### Documentation
<!--
(If it's a new feature, new test case)
Did you update documentation/Wiki relevant to your implementation?
Link to the wiki page?
-->

liat-grozovik reviewed (changes requested):
skip should be for all Spectrum platforms

liat-grozovik commented:
@congh-nvidia  i changed the subject as the description.
the skip should be only spectrum platform while it can run on other spectrum-2 + platforms.

congh-nvidia commented:
> @congh-nvidia i changed the subject as the description. the skip should be only spectrum platform while it can run on other spectrum-2 + platforms.

Hi @liat-grozovik , I've updated the skip condition and the description, please check.

liat-grozovik commented:
@bingwang-ms FYI
